jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/meta/component/StudioComponents.java

```diff
@@ -2388,12 +2388,15 @@ public interface StudioComponents {
                     @StudioProperty(xmlAttribute = "colspan", type = StudioPropertyType.INTEGER),
                     @StudioProperty(xmlAttribute = "classNames", type = StudioPropertyType.VALUES_LIST),
                     @StudioProperty(xmlAttribute = "css", type = StudioPropertyType.STRING),
-                    @StudioProperty(xmlAttribute = "forgotPasswordButtonVisible", type = StudioPropertyType.BOOLEAN),
+                    @StudioProperty(xmlAttribute = "forgotPasswordButtonVisible", type = StudioPropertyType.BOOLEAN,
+                            defaultValue = "true"),
                     @StudioProperty(xmlAttribute = "id", type = StudioPropertyType.COMPONENT_ID),
-                    @StudioProperty(xmlAttribute = "rememberMeVisible", type = StudioPropertyType.BOOLEAN),
+                    @StudioProperty(xmlAttribute = "rememberMeVisible", type = StudioPropertyType.BOOLEAN,
+                            defaultValue = "true"),
                     @StudioProperty(xmlAttribute = "enabled", type = StudioPropertyType.BOOLEAN,
                             defaultValue = "true"),
-                    @StudioProperty(xmlAttribute = "localesVisible", type = StudioPropertyType.BOOLEAN),
+                    @StudioProperty(xmlAttribute = "localesVisible", type = StudioPropertyType.BOOLEAN,
+                            defaultValue = "false"),
```

Comment:
it's still visible, even if `true` is not set explicitly. 

When you fix it, the other problem will appear that you need to set it to `true` explicitly, otherwise all existing views will be broken. Let's set it to true if `io.jmix.flowui.kit.component.loginform.EnhancedLoginForm#setLocaleItems` with not empty value to avoid breaking changes

Reply:
If we set localesVisible to true depending on the number of available locales - do we need this property at all?

loginForm.localesSelect.hidden = locales.length > 1 ? false : true;

The locales select with single item doesn't have any matter.